### PR TITLE
Added top 10 zkSync tokens

### PR DESCRIPTION
**Pull request by aalimsahin**
# Updating the Clave tokenlist

### Checklist

- [x] I am updating an existing token / project
- [x] I am adding a new token / project
- [x] I have read the [contribution guidelines](../CONTRIBUTING.md) and followed the rules accordingly

### CODE REVIEW

**Review by ulerdogan — COMMENTED** [review]
I think we should standardize the logo sizes, also mentioned as '64x64'.

File: README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@
             "name": "{NAME}",
             "symbol": "{SYMBOL}",
             "decimals": {DECIMALS},
-            "address": "{ADDRESS}",
-            "icon": "https://raw.githubusercontent.com/getclave/clave-tokenlists/master/logos/{CHAINID}_{TOKENADDRESS}.png"
+            "address": "{ADDRESS}", // lowerCase
+            "icon": "https://raw.githubusercontent.com/getclave/clave-tokenlists/master/logos/{TOKENADDRESS}.png" // lowerCase
```
Comment (ulerdogan) [inline README.md:17]: Not sure about adding illegal comments, disallows direct copy-pastes.

**Review by asgarovf — DISMISSED** [review]
Nice work. Could you also add testnet tokens please?

**Review by ulerdogan — APPROVED** [review]
g2g

**Pull request merged**